app/style/ReactSelectGlobal.less

```diff
@@ -54,21 +55,21 @@
 }
 
 .Select-control:hover:not(:disabled) {
-  box-shadow: 0 1px 0 rgba(0, 0, 0, 0.06);
-  color: @input-color-hovered;
-  border-bottom: 1px solid @stroke-color-hovered;
+  //box-shadow: 0 1px 0 rgba(0, 0, 0, 0.06);
```

Comment:
Unless you'll return this on the redesign refactor, I think you can remove the commented css.